packages/react-components/src/organisms/__tests__/CalendarList.test.tsx

```diff
@@ -1,12 +1,14 @@
 import { ComponentProps } from 'react';
-import { render } from '@testing-library/react';
+import { fireEvent, render } from '@testing-library/react';
+import { createListCalendarResponse } from '@asap-hub/fixtures';
 
 import CalendarList from '../CalendarList';
 
 const props: ComponentProps<typeof CalendarList> = {
   calendars: [],
-  page: 'calendar',
+  title: 'Subscribe to this Interest Groups Calendar',
```

Comment:
```suggestion
  title: '',
```
You don't want to include anything here that can be asserted on to encourage people to set the data within the tests for clarity,